README.md

```diff
@@ -48,19 +88,74 @@ The `execute_paginated_query` method provides a simple way to paginate the data
 
 ### Example
 ```python
+import asyncio
 from airstack.execute_query import AirstackClient
 
 api_client = AirstackClient(api_key='api-key')
 
+# Replace the below query and variables to match your use case 
+query = """
+        query CurrentTokenBalances($identity: Identity) {
+        Ethereum: TokenBalances(
+            input: {filter: {owner: {_eq: $identity}}, blockchain: ethereum, order: {lastUpdatedTimestamp: DESC}, limit: 200}
+        ) {
+            TokenBalance {
+            amount
+            tokenType
+            tokenAddress
+            token {
+                name
+            }
+            }
+            pageInfo {
+            nextCursor
+            prevCursor
+            }
+        }
+        Polygon: TokenBalances(
+            input: {filter: {owner: {_eq: $identity}}, blockchain: polygon, order: {lastUpdatedTimestamp: DESC}, limit: 200}
+        ) {
+            TokenBalance {
+            amount
+            tokenType
+            tokenAddress
+            token {
+                name
+            }
+            }
+            pageInfo {
+            nextCursor
+            prevCursor
+            }
+        }
+        }
+        """
+
+variables = {
+  "identity": "vitalik.eth"
+}
+
+# Create a Query object
 execute_query_client = api_client.create_execute_query_object(query=query, variables=variables)
 
-query_response = await execute_query_client.execute_paginated_query()
+async def get_token_balances():
+    # Execute the paginated query
+    query_response = await execute_query_client.execute_paginated_query()
+
+    # Iteratively, fetch the results from next pages
+    while query_response.has_next_page:
+        yield query_response
+        query_response = await query_response.get_next_page
+    
+    yield query_response
+
 
-if query_response.has_next_page:
-    next_page_response = await query_response.get_next_page
+async def get_all_pages():
+    responses = get_token_balances()
+    async for response in responses:
+        if not response.error: print(response.data)
 
-if next_page_response.has_prev_page:
-    prev_page_response = await query_response.get_prev_page
+asyncio.run(get_all_pages()) 
```

Comment:
Can you add one example to get the prev page

Reply:
Sure. Added in 8a86682